src/components/dialogs/reactive-limits/reactive-capability-curve/reactive-capability-curve-table.jsx

```diff
@@ -27,6 +28,17 @@ export const ReactiveCapabilityCurveTable = ({
 }) => {
     const { fields: rows, insert, remove } = useFieldArray({ name: `${id}` });
 
+    // Respect minimum size (2 rows)
+    if (rows.length < MIN_LENGTH) {
+        for (let i = 0; i < MIN_LENGTH - rows.length; i++) {
+            rows.push({
+                [MAX_Q]: null,
+                [MIN_Q]: null,
+                [P]: null,
+            });
+        }
+    }
```

Comment:
```suggestion
   const handleInsertRow = useCallback(() => {
        if (previousValues && updatePreviousReactiveCapabilityCurveTable) {
            updatePreviousReactiveCapabilityCurveTable(INSERT, rows.length - 1);
        }
        insert(rows.length - 1, {
            [P]: null,
            [MIN_Q]: null,
            [MAX_Q]: null,
        });
    }, [updatePreviousReactiveCapabilityCurveTable, previousValues, rows.length, insert]);

    // Respect minimum size (2 rows)
    useEffect(() => {
        if (rows.length < MIN_LENGTH) {
            for (let i = 0; i < MIN_LENGTH - rows.length; i++) {
                handleInsertRow();
            }
        }
    }, [rows, handleInsertRow]);
```

Reply:
Thanks i've added the useEffect only because i prefer push rather than insert